LMGRemoteData/Classes/GraphQL/Queries/Fragments.graphql

```diff
@@ -163,6 +170,29 @@ fragment LocationItem on Location {
     }
 }
 
+fragment CollectionLocationItem on Location {
+    id
+    name
+    distance
+    address {
+      id
+      city
+      state
+      centre
+    }
+}
+
+fragment LocationsLocationItem on Location {
+    id
+    name
+    address {
+      id
```

Comment:
Do we ever need address ids...? Is this used in caching / core data or something?